### PR TITLE
feat(health_checker): Daily operator balance check

### LINKED ISSUE #34: feat(health_checker): Daily operator balance check

### PR DESCRIPTION
**Pull request by toni-calvin**
Resolves #34

### CODE REVIEW

File: utils/health_checker/src/health_checker.js
Comment (ilitteri) [inline utils/health_checker/src/health_checker.js:1]: I think that we do not need to track this file

File: utils/health_checker/src/health_checker.ts
Comment (ilitteri) [inline utils/health_checker/src/health_checker.ts:1]: Missing new line at EOF

File: utils/health_checker/src/utils.ts
Comment (ilitteri) [inline utils/health_checker/src/utils.ts:1]: Ditto

**Comment by toni-calvin** [comment]
This PR can be closed as there is a PR implementing with this base but more advanced and updated. 
[PR](https://github.com/lambdaclass/zksync_stack/pull/49)

**Pull request closed without merging**